src/elements/content-sidebar/SidebarPanels.js

```diff
@@ -74,6 +79,7 @@ const MARK_NAME_JS_LOADING_DETAILS = `${ORIGIN_DETAILS_SIDEBAR}${BASE_EVENT_NAME
 const MARK_NAME_JS_LOADING_ACTIVITY = `${ORIGIN_ACTIVITY_SIDEBAR}${BASE_EVENT_NAME}`;
 const MARK_NAME_JS_LOADING_SKILLS = `${ORIGIN_SKILLS_SIDEBAR}${BASE_EVENT_NAME}`;
 const MARK_NAME_JS_LOADING_METADATA = `${ORIGIN_METADATA_SIDEBAR}${BASE_EVENT_NAME}`;
+const MARK_NAME_JS_LOADING_METADATA_REDESIGNED = `${ORIGIN_METADATA_SIDEBAR_REDESIGN}${BASE_EVENT_NAME}`;
```

Comment:
`ORIGIN_METADATA_SIDEBAR_REDESIGN` no longer exists - was previously the same as `ORIGIN_METADATA_SIDEBAR`
```suggestion
const MARK_NAME_JS_LOADING_METADATA_REDESIGNED = `${ORIGIN_METADATA_SIDEBAR}${BASE_EVENT_NAME}`;
```

Reply:
After further consideration I decided to leave `ORIGIN_METADATA_SIDEBAR_REDESIGN` and apply it properly, so we can track the loading performance. I'm sure @jstoffan knows how.